src/github.com/getlantern/chained/dialer.go

```diff
@@ -88,7 +89,7 @@ func checkCONNECTResponse(r *bufio.Reader, req *http.Request) error {
 	if err != nil {
 		return fmt.Errorf("Error reading CONNECT response: %s", err)
 	}
-	if resp.StatusCode < 200 || resp.StatusCode > 299 {
+	if resp.StatusCode < http.StatusOK || resp.StatusCode > 299 {
```

Comment:
I think anyone reading the Lantern source better be familiar with the fact that status codes correspond to integers.  I don't know them all by heart, though, so I like your constants.  If the awkwardness of mixing numbers and constants bothers you, you can hide it in a function, like

go```
func sameStatusCodeClass(statusCode1 int, statusCode2 int) {
    // HTTP response status "classes" come in ranges of 100
   classRange = 100
    return statusCode1 % classRange == statusCode2 % classRange
}
```

Then you can make that check like

go```
if !sameStatusCodeClass(resp.StatusCode, http.StatusOK) {
   ...
}
```